code/modules/transport/pods/MainWeapon.dm

```diff
@@ -380,6 +380,624 @@
 		opencomputer(usr)
 		return
 
+// Extra-Facility Integrity Fabricator: the big boyo pod construction tool
+
+//mode defines for tool
+#define EFIF_MODE_DRILL 0
```

Comment:
Is there a reason why these can drill too? It seems a little out of place and these are QM buyable compared to the regular mining drills requiring bohrum and uqil.

Reply:
Lets them work effectively for asteroid field construction (one of the fun use cases I had in mind). I can dial back the fire rate to something similar to the industrial utility arms or up the QM price, if you like.